### PR TITLE
Fix for out of memory error issue #280

### PR DESCRIPTION
**Pull request by sdv4**
The problem was fixed by removing the creation of an array containing a vector for every image file in utils.py/get_ImageDataGenerator. The code was there in preparation for a call to fit() for the ImageGenerator  object. Upon inspecting the documentation for this method, it turns out that this call was not necessary for the image processing we are doing. 

I tested this on Colab for a few epochs on vgg16 using imagenet weights.

### CODE REVIEW

**Comment by marco-c** [comment]
I wonder why this wasn't a problem in @sagarvijaygupta case (he was using Colab too).

> Upon inspecting the documentation for this method, it turns out that this call was not necessary for the image processing we are doing.

It isn't necessary for the image processing we are doing right now, but could be needed in the future if we try to do some. Anyway, we can bring it back if needed.

File: autowebcompat/utils.py [hunk 1]
```diff
@@ -81,20 +81,12 @@ def prepare_images():
 
 
 def load_image(fname, parent_dir='data_resized'):
-    img = load_img(os.path.join(parent_dir, fname), target_size=(32, 24))
+    img = load_img(os.path.join(parent_dir, fname), target_size=(224, 224))
```
Comment (marco-c) [inline autowebcompat/utils.py]: Can you remove this change from the PR? It should be done in a separate PR, after this one.

File: label_persons/data [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Image Name,Label
```
Comment (marco-c) [inline label_persons/data]: Can you remove this file from the PR?

**Comment by sdv4** [comment]
I am going to open a new pull request that with a clean commit history and with the changes @marco-c specified above.

**Comment by marco-c** [comment]
> Was he using Colab when the dataset was much smaller, before all of that additional labelling was done back in Oct 2018?

Yes, I think so.

> I am going to open a new pull request that with a clean commit history and with the changes @marco-c specified above.

It's better not to close a PR and open a new one in cases like this (well, basically in all cases :P). Otherwise you lose the history of discussion.

**Comment by sdv4** [comment]
> It's better not to close a PR and open a new one in cases like this (well, basically in all cases :P). Otherwise you lose the history of discussion.

Oh ok, that makes sense. Thanks for letting me know; I will make sure to follow this procedure in the future!
